analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -158,7 +158,7 @@ class Pip(
     repoConfig: RepositoryConfiguration
 ) : PackageManager(name, analysisRoot, analyzerConfig, repoConfig), CommandLineTool {
     class Factory : AbstractPackageManagerFactory<Pip>("PIP") {
-        override val globsForDefinitionFiles = listOf("requirements*.txt", "setup.py")
+        override val globsForDefinitionFiles = listOf("*requirements*.txt", "setup.py")
```

Comment:
At first I thought to vaguely remember that we deliberately didn't do this in the past it would lead to problems with clashes of derived project names in case of multiple `*requirements*.txt` in the same directory, of with other code in `Pip.kt` that assumes the file is always called `requirements.txt`, but it seems all these issues were resolved by now.